hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/ddl/QueryBasedDDLExecutor.java

```diff
@@ -220,5 +221,16 @@ private List<String> constructChangePartitions(String tableName, List<String> pa
     }
     return changePartitions;
   }
+
+  /**
+   * SQL statement should be be escaped in order to consider anti-slash
+   *
```

Comment:
Every sentence should end up with `.`.  Every new paragraph should start with `<p>`, remove the params comments if there are no comments at all.